openff/toolkit/tests/test_parameters.py

```diff
@@ -2092,95 +2092,6 @@ class TestVirtualSiteHandler:
     Test the creation of a VirtualSiteHandler and the implemented VirtualSiteTypes
     """
 
-    @classmethod
```

Comment:
Note to self: See if there's a better way to satisfy mypy other than the removals in https://github.com/openforcefield/openff-toolkit/pull/1285/commits/59598995dd889b284dea5e982f3d8a6d970fcd54